README.adoc

```diff
@@ -116,21 +117,21 @@ You'll apply various annotations to the properties and declaration of a Java cla
 
 The [hotspot=class file=0]`JavaInfo` class is annotated with an `@Type` annotation. 
 The [hotspot=type file=0]`@Type("java")` annotation maps this class to define the `java` object type in GraphQL.
-The `java` object type will give information on the system's Java installation. 
+The `java` object type gives information on the system's Java installation. 
```

Comment:
Change "the system's Java installation" to "the Java installation of the system" to avoid giving possessive traits to entities that aren't referring to users.